packages/scripts/templates/test/${featureName.dashCase}.spec.tsx.tmpl

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai';
```

Comment:
import { expect } from chai.
but again.. I am not sure we want to force the user our method of testing.. he should be able to write his test files ni any method he wants, not specific to 'expect'. I would remove this template